src/SonarScanner.Shim/PropertiesFileGenerator.cs

```diff
@@ -89,17 +89,23 @@ public bool TryWriteProperties(PropertiesWriter writer, out IEnumerable<ProjectD
         {
             var projects = ProjectLoader.LoadFrom(analysisConfig.SonarOutputDir);
 
-            if (projects == null || !projects.Any())
+            if (projects == null ||
```

Comment:
Knowing the implementation of `ProjectLoader.LoadFrom()` the check for `null` is redundant...